docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -401,18 +401,12 @@ const buildComponentDocumentation = async (options: {
         typeDescriptions,
       };
 
-      const jsdocDefaultValue = getJsdocDefaultValue(
+      const defaultValue = getJsdocDefaultValue(
```

Comment:
This is named `jsdocDefaultValue` because in the core script, they also check the default props according to the props deconstruction (`const { color = 'inherit', disabled = false } = props` will add default value to `docs:api`

Reply:
> By the way, https://github.com/mui/mui-x/pull/11303 will remove this file but keep the new behavior you defined.

Should I proceed with this PR then? It's not urgent at all, so if it's already addressed by https://github.com/mui/mui-x/pull/11303 I can close this one 🙂

Reply:
You can merge it, it will reduce the gif diff :)